README.md

```diff
@@ -6,16 +6,23 @@
 
 OpenSearch Security is a plugin for OpenSearch that offers encryption, authentication and authorization. When combined with OpenSearch Security-Advanced Modules, it supports authentication via Active Directory, LDAP, Kerberos, JSON web tokens, SAML, OpenID and more. It includes fine grained role-based access control to indices, documents and fields. It also provides multi-tenancy support in OpenSearch Dashboards.
 
-- [Features](#features)
-- [Installation](#installation)
-- [Test and Build](#test-and-build)
-- [Config hot reloading](#config-hot-reloading)
-- [Contributing](#contributing)
-- [Getting Help](#getting-help)
-- [Code of Conduct](#code-of-conduct)
-- [Security](#security)
-- [License](#license)
-- [Copyright](#copyright)
+- [OpenSearch Security Plugin](#opensearch-security-plugin)
+  - [Features](#features)
+    - [Encryption](#encryption)
+    - [Authentication](#authentication)
+    - [Access control](#access-control)
+    - [Audit/Compliance logging](#auditcompliance-logging)
```

Comment:
`audit/compliance-logging` instead?

Reply:
why? This is auto-generated with `prettier`

Reply:
The format `auditcompliance-logging` looks different from other items. If only this format will work, then let's go with it.